uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -576,4 +578,13 @@ private Mqtt5Message buildMqtt5Message(int qos, boolean retain, @NonNull String
                             .setRetain(retain)
                             .build();
     }
+
+    private int getFreePort() {
```

Comment:
Why that is required ?

Reply:
Removed b1d5d068b6318084cd85a83b605cb7067c5558a6